src/RenderDocs.hs

```diff
@@ -18,58 +18,68 @@ import Obj
 import Types
 import Util
 
-saveDocsForEnvs :: FilePath -> String -> [(SymPath, Env)] -> IO ()
-saveDocsForEnvs dirPath projectTitle envs =
-  let allEnvNames = (fmap (getModuleName . snd) envs)
-  in  do mapM_ (saveDocsForEnv dirPath projectTitle allEnvNames) envs
-         writeFile (dirPath ++ "/" ++ projectTitle ++ "_index.html") (projectIndexPage projectTitle allEnvNames)
-         putStrLn ("Generated docs to '" ++ dirPath ++ "'")
+saveDocsForEnvs :: Project -> [(SymPath, Env)] -> IO ()
+saveDocsForEnvs ctx envs =
+  let dir = projectDocsDir ctx
+      title = projectTitle ctx
+      allEnvNames = (fmap (getModuleName . snd) envs)
+  in  do mapM_ (saveDocsForEnv ctx allEnvNames) envs
+         writeFile (dir ++ "/" ++ title ++ "_index.html") (projectIndexPage ctx allEnvNames)
+         putStrLn ("Generated docs to '" ++ title ++ "'")
```

Comment:
```suggestion
         putStrLn ("Generated docs to '" ++ dirPath ++ "'")
```

Reply:
Whoops, you’re right of course!

Reply:
It can't find this variable so the build fails. Where is it defined?